keps/349-avoid-coscheduling/README.md

```diff
@@ -0,0 +1,350 @@
+# KEP-349: Avoid concurrent pod scheduling for workloads
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [KEP-349: Avoid concurrent pod scheduling for workloads](#kep-349-avoid-concurrent-pod-scheduling-for-workloads)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [User Stories (Optional)](#user-stories-optional)
+      - [Story 1](#story-1)
+      - [Story 2](#story-2)
+    - [Notes/Contraints/Caveats (Optional)](#notescontraintscaveats-optional)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Design Details](#design-details)
+    - [Ready workload condition](#ready-workload-condition)
+    - [Workload API](#workload-api)
+    - [Job annotations](#job-annotations)
+      - [Wait for pod scheduling](#wait-for-pod-scheduling)
+      - [Pod scheduled timeout seconds](#pod-scheduled-timeout-seconds)
+    - [Test Plan](#test-plan)
+        - [Prerequisite testing updates](#prerequisite-testing-updates)
+      - [Unit Tests](#unit-tests)
+      - [Integration tests](#integration-tests)
+    - [Graduation Criteria](#graduation-criteria)
+  - [Implementation History](#implementation-history)
+  - [Drawbacks](#drawbacks)
+  - [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This proposal introduces a mechanism for avoid co-scheduling of workload pods.
```

Comment:
co-scheduling is already a term that we use somewhere else.

```suggestion
This proposal introduces a mechanism to avoid concurrent scheduling of pods from different workloads,
if they require all-or-nothing scheduling guarantees.
```